openformats/utils/icu.py

```diff
@@ -6,28 +6,53 @@
 from openformats.exceptions import ParseError
 
 
+NUMERIC_RULES = ['=0', '=1', '=2']
+SUPPORTED_PLURAL_RULES = Handler._RULES_ATOI.keys() + NUMERIC_RULES
+
+
+def normalize_plural_rule(rule_str):
+    """Returns the equivalent rule name as 'one', 'two', etc.
+
+    ICU supports plurality rules in the form of =N (e.g. =2).
+    This method converts any such string to its equivalent "word" form.
+    >>> normalize_plural_rule('=0') == 'zero'
+    >>> normalize_plural_rule('zero') == 'zero'
+
+    :param str rule_str: the name of the plural rule, e.g. '=1' or 'zero'
+    :return: the word form of the plural rule
+    :rtype: str
+    """
+    if rule_str == '=0':
```

Comment:
I think it would be better to create a dictionary:
```suggestion
    rule_mapping = {
        '=0': 'zero',
        '=1': 'one',
        '=2': 'two',
    }
    return rule_mapping.get(rule_str, rule_str)
```